### PR TITLE
chore: refs T33926 fix votes

### PR DESCRIPTION
**Pull request by mussbach**
**Ticket:** https://yaits.demos-deutschland.de/T33926 

On adding a new StatementVote to a statement the StatementVote was only added to the database, not to the statement itself. Because of that the elasticsearch index was not updated and this triggered a visual bug in the interface as it seemed that the Statement could be voted again.
Along the way `publicVerified` was once refactored to be a string, not a boolean, two occurrences where missing.

This is an alternate approach to #1784 

### How to review/test
Vote to a statement as a citizen

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Comment by jalalifardemos** [comment]
It works for Private Person, but not for Owner

**Comment by mussbach** [comment]
What do you mean by owner?

**Comment by jalalifardemos** [comment]
Owner of Procedure

File: demosplan/DemosPlanCoreBundle/Logic/Statement/StatementHandler.php [hunk 1]
```diff
@@ -1913,7 +1913,7 @@ public function getStatementCounts($procedureId, $role, $user): array
         if (Role::CITIZEN === $role) {
             $public = $this->statementService->getStatementsByProcedureId(
                 $procedureId,
-                ['publicVerified' => true],
+                ['publicVerified' => Statement::PUBLICATION_APPROVED],
                 null,
                 null,
                 1_000_000
```
[the remark this answers is not in the document]
Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Logic/Statement/StatementHandler.php:1919]: This is only a drive-by fix as publicVerified was refactored some time ago.

**Review by breierDemos — APPROVED** [review]

**Pull request merged**